### PR TITLE
More content edits

### PR DESCRIPTION
**Pull request by tdlowden**

### CODE REVIEW

File: app/templates/views/accessibility_statement.html [hunk 1]
```diff
@@ -166,11 +158,11 @@ <h3 class="heading-small" id="pdfs-documents">
     We provide a written description of the information in the PDF on an HTML page as an accessible alternative.
   </p>
 
-  <h3 class="heading-small" id="emails">
+ <!-- <h3 class="heading-small" id="emails">
     Emails
   </h3>
 
-  <p class="govuk-body">
+  <<p class="govuk-body">
```
Comment (rahearn) [inline app/templates/views/accessibility_statement.html:165]: [issue (nonblocking)] accidental double `<` 

```suggestion
  <p class="govuk-body">
```

**Review by rahearn — COMMENTED** [review]
To the question about when to update the a11y findings, I think we should table the things mentioned until we replace the UK design system with USWDS

File: app/templates/views/accessibility_statement.html [hunk 2]
```diff
@@ -152,7 +144,7 @@ <h3 class="heading-small" id="problems-with-using-the-interface">Problems with u
   </ul>
 
   <p class="govuk-body">
-    We hope to fix these issues by the end of September 2022.
+    We hope to fix these issues by the end of March 2023.
```
Comment (stvnrlly) [inline app/templates/views/accessibility_statement.html:147]: Same here. What's the basis for this estimate?

File: app/templates/views/privacy.html [hunk 3]
```diff
@@ -150,7 +135,7 @@ <h2 class="heading-medium" id="questions-and-complaints">Questions and complaint
     Water Lane<br>
     Wilmslow<br>
     Cheshire SK9 5AF
-    </p>
+    </p> -->
```
Comment (stvnrlly) [inline app/templates/views/privacy.html:138]: Should we just delete this part?

File: app/templates/views/accessibility_statement.html [hunk 4]
```diff
@@ -13,20 +13,20 @@ <h1 class="heading-large">Accessibility statement</h1>
...
-      "Last updated": "27 April 2022",
-      "Next review due": "21 June 2022"
+      "Last updated": "21 November 2022",
+      "Next review due": "21 February 2023"
     }
     ) }}
 
   <p class="govuk-body">
-    This accessibility statement applies to the us notify domain. It does not apply to the <a class="govuk-link govuk-link--no-visited-state" href="https://docs.notifications.service.gov.uk">US Notify API documentation subdomain</a>.
+    This accessibility statement applies to the US notify domain. <!-- It does not apply to the <a class="govuk-link govuk-link--no-visited-state" href="https://docs.notifications.service.gov.uk">US Notify API documentation subdomain</a>.-->
```
Comment (stvnrlly) [inline app/templates/views/accessibility_statement.html:22]: We probably want to capitalize "Notify" here

**Review by stvnrlly — COMMENTED** [review]
A couple of small comments, but it looks good.

There's also an implicit idea here that we want to use the pre-existing page structure. If we're going to base this on another service, though, we may want to adopt something like https://login.gov/policy/ and https://login.gov/accessibility/ as our starting point. That doesn't need to hold up this PR, but is something for us to think about as we edit this further.

**Comment by tdlowden** [comment]
To @stvnrlly 's point- that is definitely something that I became aware of while doing this. I think essentially this first round of edits can serve as a content base from which to pull from when we think about the restructuring, esp after we incorporate USWDS. I don't really have the tech chops to start redesigning pages, but I agree that we should think about it all.

File: app/templates/views/accessibility_statement.html [hunk 5]
```diff
@@ -124,7 +116,7 @@ <h3 class="heading-small" id="non-compliance">Non compliance with the accessibil
   </ul>
 
   <p class="govuk-body">
-    We expect to fix these issues by the end of July 2022.
+    We expect to fix these issues by the end of March 2023.
```
[the remark this answers is not in the document]
Comment (tdlowden) [reply app/templates/views/accessibility_statement.html:119]: good point. removed.

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**